ncurses/PKGBUILD

```
@@ -71,8 +72,8 @@ build() {
     --with-build-cflags=-D_XOPEN_SOURCE_EXTENDED \
     --with-pkg-config-libdir=/usr/lib/pkgconfig
 
-  make
-  make DESTDIR=${srcdir}/dest install
+  make -j$(nproc)
+  make -j$(nproc) DESTDIR=${srcdir}/dest install
```

```suggestion
  make
  make DESTDIR=${srcdir}/dest install
```

we set that via MAKEFLAGS already